src/modules/leaderboardAPI/getScores.js

```diff
@@ -0,0 +1,9 @@
+export default async (ID = 'yVWfxt83ZzUTJiUF6itj') => {
+  const requestURL = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${ID}/scores`;
+  const request = new Request(requestURL);
+
+  const response = await fetch(request);
+  const LB = await response.json();
+
+  return LB.result;
+};
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly data disappear when the page is updated, all users given dates should be saved 🙏🏼 

![mobile](https://github.com/Salimer/Leaderboard-project/assets/63915024/ef3e1663-ca68-4450-bf99-ea4534c2db86)